### ISSUE TITLE
Set anti-FLoC server header

### ISSUE THREAD

**Issue by josaur** [humhub/humhub#5008]
**Pitch**

Humhub servers should set the Permissions-Policy response header as follows: Permissions-Policy: interest-cohort=() (https://github.com/WICG/floc#opting-out-of-computation). It disables the FLoC tracking method in Chrome browsers for users that visit the Humhub Instance.

**Motivation**
Currently, the new Google tracking method called FLoC is in the media. It cannot be deactivated by Chrome users easily. However, site admins can disable FLoC for their sites by sending the Permissions-Policy header with correct value.
If Humhub servers send them by default, this has a huge privacy impact for all Humhub users.

**Comment by Seirdy** [comment]
(Copying this a comment from an identical issue elsewhere).

Relevant discussion: https://github.com/friendica/friendica/issues/10151

TLDR: it's probably not a good idea to include this header in upstream software.
